src/isomorphic/classic/class/__tests__/ReactClassMixin-test.js

```diff
@@ -368,24 +372,24 @@ describe('ReactClass-mixin', function() {
   });
 
   it('should include the mixin keys in even if their values are falsy',
-    function() {
-      var mixin = {
-        keyWithNullValue: null,
-        randomCounter: 0
-      };
+      function() {
```

Comment:
I dunno. I think it would be best if you'd get a lint warning on new lines that are too long but that we could still commit long lines if we wanted to. That's hard to make work with our current setup though.

I like the line length limit for reading actual code but it doesn't feel useful to me on lines like this.